### ISSUE TITLE
Undefined names: banner_begin(), banner_finished(), banner_new()

### ISSUE THREAD

**Issue by cclauss** [zsdlove/ApkVulCheck#1]
[flake8](http://flake8.pycqa.org) testing of https://github.com/zsdlove/ApkVulCheck on Python 3.7.0

$ __flake8 . --count --select=E901,E999,F821,F822,F823 --show-source --statistics__
```
./AndroidCodeCheck.py:94:2: F821 undefined name 'banner_begin'
	banner_begin()
 ^
./AndroidCodeCheck.py:112:2: F821 undefined name 'banner_begin'
	banner_begin()
 ^
./AndroidCodeCheck.py:113:2: F821 undefined name 'banner_finished'
	banner_finished()
 ^
./AndroidCodeCheck.py:382:2: F821 undefined name 'banner_new'
	banner_new()
 ^
4     F821 undefined name 'banner_begin'
4
```

**Comment by zsdlove** [comment]
注释掉即可，没什么用

**Comment by cclauss** [comment]
Fixed in #2
